### PR TITLE
[Tests][API] More test coverage for Appointments and Cages

### PR DESCRIPTION
**Pull request by Al-Andrew**

### CODE REVIEW

File: src/DucksNet.API/Controllers/AppointmentsController.cs [hunk 1]
```diff
@@ -31,19 +31,12 @@ public async Task<IActionResult> GetByPetID(Guid petId)
         var appointments = await _appointmentsRepository.GetAllAsync();
         return Ok(appointments.Where(a => a.PetId == petId));
     }
-
-    [HttpGet("byEmployee/{employeeId}")]
-    public async Task<IActionResult> GetByVetID(Guid vetId)
-    {
-        var appointments = await _appointmentsRepository.GetAllAsync();
-        return Ok(appointments.Where(a => a.VetId == vetId));
```
Comment (OpariucRares) [inline src/DucksNet.API/Controllers/AppointmentsController.cs:39]: why did you remove it?

File: tests/DucksNet.IntegrationTests/AppointmentsControllerTests.cs [hunk 2]
```diff
@@ -155,22 +214,6 @@ private async Task<Guid> SetupPet(Guid ownerId ,string petSizeString)
         return pet!.Id;
     }
 
-    private async Task<Guid> SetupCage(Guid officeId, string size)
-    {        
-        var sut = new CreateCageDto {
```
Comment (OpariucRares) [inline tests/DucksNet.IntegrationTests/AppointmentsControllerTests.cs:160]: why did you remove it?

**Review by OpariucRares — CHANGES_REQUESTED** [review]
comments

File: tests/DucksNet.IntegrationTests/AppointmentsControllerTests.cs (hunk 2 above)
Comment (Al-Andrew) [reply tests/DucksNet.IntegrationTests/AppointmentsControllerTests.cs:160]: Because it wasn't used in the tests and it appeared as dead code in the coverage

File: src/DucksNet.API/Controllers/AppointmentsController.cs (hunk 1 above)
Comment (Al-Andrew) [reply src/DucksNet.API/Controllers/AppointmentsController.cs:39]: Because we don't support this functionality yet :) Assigning a vet to an appointment

**Review by OpariucRares — APPROVED** [review]
the functionality should be supported, but because of lack of time, I will mark it as ok :shipit:
